docs/website/docs/plus/features/projects.md

```diff
@@ -250,8 +356,91 @@ pipeline = entities.create_pipeline("bronze_pipe")
 transformation = entities.create_transformation("stressed_transformation")
```

Comment:
Can you replace this line with create_destination like in Dave's original tutorial?